Assets/Scripts/UI/Interaction Wheel/WheelButton.cs

```diff
@@ -0,0 +1,50 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+using System.Collections;
+
+
+public class WheelButton : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler {
+
+    public Image circle;
+	public Image icon;
```

Comment:
What happened to the alignment here >.> (Also, again CamelCase for public fields & properties)